Web/src/components/TransactionFilterDrawer.jsx

```
@@ -0,0 +1,188 @@
+                {/* Amount Filter */}
+                <MUIGrid item xs={12} md_1={3}>
+                    <MUITypography variant="h6" gutterBottom textAlign={'center'}>Amount Range</MUITypography>
+                    <MUIBox display="flex" gap={2}>
+                        <MUITextField 
+                            label="Min Amount" 
+                            type="number"
+                            inputProps={{ step: "0.01" }}
+                            value={amountFilter.min}
+                            onChange={(e) => {
+                                const value = parseFloat(e.target.value);
+                                if (!isNaN(value)) { 
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)